QTeachersApp/View/create.cpp

```diff
@@ -0,0 +1,99 @@
+        sex = ui->lineEdit_sex->text();
+        pesel = ui->lineEdit_pesel->text();
+        dateOfBirth = ui->lineEdit_dateOfBirth->text();
+        title = ui->lineEdit_title->text();
+        listOfSubjects = ui->lineEdit_listOfSubjects->text();
+        // Getting data from lineEdits of ui - END OF CODE
+        if((*myTeachersDatabase).isOpen())
+        {
+            qInfo("I'm in creation mode right now!");
+            // Checking if there is the same data in database already - START OF CODE
+            QSqlQuery querySelect;
+            querySelect.prepare("SELECT * FROM Teachers");
+            querySelect.exec();
+            bool checkForEqualisation = false; // if the same data (with only diffrence being another "ID") are already in teachers.sqlite database
+            while(querySelect.next())
+            {
+                   if(name == querySelect.value(1).toString()
+                      && surname == querySelect.value(2).toString()
+                      && sex == querySelect.value(3).toString()
+                      && pesel == querySelect.value(4).toString()
+                      && dateOfBirth == querySelect.value(5).toString()
+                      && title == querySelect.value(6).toString()
+                      && listOfSubjects == querySelect.value(7).toString())
+                    {
+                        checkForEqualisation = true;
+                        qDebug() << "Creating your new \"teacher\" will be imposssible, because he already is in our database (only with another id).";
+                    }
+
+            }
+            if(!dataValidation(name, surname, sex, pesel, dateOfBirth, title, listOfSubjects) || checkForEqualisation)
+            {
+
+                clearingLineEdits();
+                return;
+            }
+            // Checking if there is the same data in database already - END OF CODE
+            // Creation of new teacher - START OF CODE
+            QSqlQuery queryCreate;
+            queryCreate.prepare("INSERT INTO Teachers (Name, Surname, Sex, PESEL, DateOfBirth, Title, ListOfSubjects) VALUES (?, ?, ?, ?, ?, ?, ?)");
+            queryCreate.addBindValue(name);
+            queryCreate.addBindValue(surname);
+            queryCreate.addBindValue(sex);
+            queryCreate.addBindValue(pesel);
+            queryCreate.addBindValue(dateOfBirth);
+            queryCreate.addBindValue(title);
+            queryCreate.addBindValue(listOfSubjects);
+            if(queryCreate.exec())
+            {
+                qInfo() << "Everything is working and you querry was added to your teachers database. :)";
+            }
+            else
+            {
+                qDebug() << "Something went terribly wrong.";
+            }
+            // Creation of new teacher - END OF CODE
+
+         }
+         clearingLineEdits();
+     }
+
+     void Create::clearingLineEdits() // function is making line edits "empty" after all operations, whatever they are
```

Comment:
https://google.github.io/styleguide/cppguide.html#Implementation_Comment_Donts

Generalnie staramy się tworzyć samoopisujący się kod, tzn. nie wstawiamy komentarzy, które nic nie wnoszą, bo opisują rzeczy oczywiste - tylko brudzą - za wyjątkiem wymagań dokumentacji. 
W tym przypadku niektórzy używają tzw. javadoców, to są komentarze nad metodami, które zawierają w sobie m.in. opis argumentów (@param) i co dana metoda zwraca (@returns), przydatne do pracy w zespole i generowania dokumentacji. W C++ widzę, że odpowiednikiem jest np. https://www.doxygen.nl/index.html.